### ISSUE TITLE
WakaTime imports do not consider all Machine/UserAgent entries

### ISSUE THREAD

**Issue by jabra98** [muety/wakapi#288]
It seems that WakaTime's API at `/api/v1/users/current/machine_names` and `/api/v1/users/current/user_agents` only show the first 100 entries respectively. Inside the JSON response there is some metadata, in particular one field describes how many pages there are (each of which shows at most 100 entries). 

I figured that the fetch functions https://github.com/muety/wakapi/blob/aaa907a7b281055cb6040ccb497f7536f8f410b3/services/imports/wakatime.go#L174 and https://github.com/muety/wakapi/blob/aaa907a7b281055cb6040ccb497f7536f8f410b3/services/imports/wakatime.go#L202 only concern themselves with the first page (the default one), hence it may happen as it did in my case, that the code here https://github.com/muety/wakapi/blob/aaa907a7b281055cb6040ccb497f7536f8f410b3/services/imports/wakatime.go#L240 and here https://github.com/muety/wakapi/blob/aaa907a7b281055cb6040ccb497f7536f8f410b3/services/imports/wakatime.go#L248 inserts wrong data.

I wrote a dirty patch that resolves this issue by simply looping over the pages in a hardcoded fashion
```Go
func (w *WakatimeHeartbeatImporter) fetchMachineNames() (map[string]*wakatime.MachineEntry, error) {
	httpClient := &http.Client{Timeout: 10 * time.Second}

	machines := make(map[string]*wakatime.MachineEntry)
        
        // the WakaTime API has 8 Pages for me
        for i:=1; i<9; i++ {
            req, err := http.NewRequest(http.MethodGet, config.WakatimeApiUrl+config.WakatimeApiMachineNamesUrl+"?page="+strconv.Itoa(i), nil)
            if err != nil {
                    return nil, err
            }

            res, err := httpClient.Do(w.withHeaders(req))
            if err != nil {
                    return nil, err
            }

            var machineData wakatime.MachineViewModel
            if err := json.NewDecoder(res.Body).Decode(&machineData); err != nil {
                    return nil, err
            }

            for _, ma := range machineData.Data {
                    machines[ma.Id] = ma
            }
        }

	return machines, nil
}
``` 
and similarly for the other function. Surely this is easily fixable by considering the metadata in the GET response, but as I don't know Go too well I figured I just report the issue here instead of writing up a PR.

**Comment by muety** [comment]
Good catch, thanks for bringing this up and for your first sketch. What you came up with is already half the battle. The only thing missing is to extend `MachineViewModel` (and `UserAgentsViewModel, respectively) to contain fields for pagination and then read them in your currently hard-coded loop.

Approximately like this:
```go
type UserAgentsViewModel struct {
	Data       []*UserAgentEntry `json:"data"`
	Total      int               `json:"total"`
	TotalPages int               `json:"total_pages"`
}
```

If you feel confident to implement this last missing piece, feel free to open a PR. Otherwise let me know, then I'll have a look as soon as I can.

**Comment by muety** [comment]
Resolved by #290
